infinispan-spring-boot-starter/src/test/java/test/infinispan/autoconfigure/CacheManagerTest.java

```diff
@@ -1,43 +1,41 @@
 package test.infinispan.autoconfigure;
 
+import org.infinispan.client.hotrod.RemoteCacheManager;
+import org.infinispan.manager.DefaultCacheManager;
 import org.infinispan.spring.starter.embedded.InfinispanEmbeddedAutoConfiguration;
 import org.infinispan.spring.starter.embedded.InfinispanEmbeddedCacheManagerAutoConfiguration;
 import org.infinispan.spring.starter.remote.InfinispanRemoteAutoConfiguration;
 import org.infinispan.spring.starter.remote.InfinispanRemoteCacheManagerAutoConfiguration;
-import org.infinispan.client.hotrod.RemoteCacheManager;
-import org.infinispan.manager.DefaultCacheManager;
 import org.junit.Assert;
-import org.junit.Test;
-import org.junit.runner.RunWith;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.context.ApplicationContext;
-import org.springframework.test.annotation.DirtiesContext;
-import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
 
-@DirtiesContext
```

Comment:
I take it that this and the other `@DirtiesContext` annotations that were removed were not need in the first place?

Reply:
I'm going to do a pass on the tests to check properly this because I removed checking if all the tests passed properly but I might be doing something wrong here!